src/features/add-hectares.feature

```diff
@@ -0,0 +1,21 @@
+@regression
+Feature: BNGP-786 Enter Area in hectares
+
+    As a Site Provider
+    I need to enter the total hectares of my site
+    So the Operator can identify the full size of the site
+
+    Background:
+        Given I am on the "add-hectares" page
+
+    @e2e
+    Scenario Outline: I can add the total hectares for my land
+        Then I should be able to add my total hectares of "<hectares>"
```

Comment:
Don't you want to assert that the value has been added to the submission? this just checks you can submit a form with that value.